frontend/src/components/landing/carousel/ContentCards.module.scss

```diff
@@ -0,0 +1,85 @@
+@import "../../../styles/tokens.scss";
+@import "~@mozilla-protocol/core/protocol/css/includes/lib";
+
+.wrapper {
```

Comment:
It's a bit hard to see since there's no Figma, but I feel like there might need to be a little padding added to `.wrapper`? (But maybe check with design - you can just send them [the Netlify link](https://deploy-preview-1870--fx-relay-demo.netlify.app/#use-cases/offline).)